.github/workflows/nightly.yml

```diff
@@ -0,0 +1,60 @@
+name: "Continuous Integration (Nightly)"
+
+on:
+  schedule:
+    - cron: "12 3 * * *"
+
+env:
+  fail-fast: true
+
+jobs:
+  phpunit-mariadb:
+    name: "PHPUnit with MariaDB"
+    runs-on: "ubuntu-24.04"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.4"
+          - "8.3"
+        mariadb-version:
+          - "earliest"
```

Comment:
If the goal of this workflow is to catch issues with the latest versions of mariadb, why would you include `earliest`? Or maybe I misunderstand what `earliest` means here?

Reply:
Earliest is there so that if we make a mistake fixing a bug in the earliest version it will be noticed. Hopefully the probability is low.